app/views/dashboard/ng-partials/widgets/_build_sequence.html.haml

```diff
@@ -1,35 +1,29 @@
 .widget-wrapper
   .row
     .small-12.columns
+      %button.green.button-like{ "ng-click" => "saveSequence(sequence)", style: "margin: 9px;" }
+        Save
+      %button.yellow.button-like{ "ng-click" => "execute(sequence)", style: "margin: 9px;" }
+        Execute
+      %button.red.button-like{ "ng-click" => "deleteSequence(sequence)", style: "margin: 9px;" }
+        Delete
       .header-wrapper
-        %h5 Build A Sequence
+        %h5 Sequence Builder
   .row
     .small-12.columns
       .content-wrapper
         .row
-          .small-6.columns
-            %input#right-label{"ng-model" => "sequence.name", :type => "text"}
-          .small-2.columns
-            %button.green.button-like{"ng-click" => "saveSequence(sequence)"}
-              Save
-          .small-2.columns
-            %button.yellow.button-like{"ng-click" => "execute(sequence)"}
-              Execute
-          .small-2.columns
-            %button.red.button-like{"ng-click" => "deleteSequence(sequence)"}
-              Delete
-        .row
           .small-12.columns
-            .drag-drop-area.padding
-              %h6 DRAG BASIC OPERATIONS AND/OR SAVE SEQUENCES HERE
+            %label.left NAME
+            %input{"ng-model" => "sequence.name", :type => "text"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace